src/blocker.rs

```diff
@@ -1346,14 +1389,13 @@ mod legacy_rule_parsing_tests {
     // easyList = { 24478, 31144, 0, 5589 };
     // not handling (and not including) filters with the following options: 
     // - $popup
-    // - $generichide
     // - $subdocument
```

Comment:
not about this PR, but I think we actually do handle `$subdocument`

Reply:
looks like `$document` right below is handled too. I'll go ahead and remove both.

Reply:
`$document` only somewhat - there is a spec for handling main document url blocking, but not implemented

Reply:
ah oops, I'll add that back then :sweat_smile: